experimental/FTheoryTools/src/LiteratureModels/attributes.jl

```diff
@@ -420,6 +420,28 @@ function model_description(m::AbstractFTheoryModel)
   return get_attribute(m, :model_description)
 end
 
+@doc raw"""
+    model_parameters(m::AbstractFTheoryModel)
+
+Return the `model_parameters` of the given model.
+If no `model_parameters` are known, an error is raised.
+
+```jldoctest
+julia> m = literature_model(arxiv_id = "1212.2949", equation = "3.2", model_parameters = Dict("k" => 5))
+Assuming that the first row of the given grading is the grading under Kbar
+
+Global Tate model over a not fully specified base -- SU(2k+1) Tate model with parameter values (k = 5) based on arXiv paper 1212.2949 Eq. (3.2)
+
+julia> model_parameters(m)
+Dict{String, Int64} with 1 entry:
+  "k" => 5
+```
```

Comment:
Very good. This here is exactly what I was looking for earlier, but somehow must have overlooked this test. Nice!